recipe/meta.yaml

```diff
@@ -11,32 +11,33 @@ source:
 
 build:
   number: 0
-  skip: true  # [py<39]
-  script: {{ PYTHON }} -m pip install . -vv
+  # maturin isn't available on s390x
+  skip: true  # [py<39 or s390x]
```

Comment:
Maybe we should change this to `[py<37]` to correspond with upstream

https://github.com/laspy/laspy/blob/2.4.1/setup.py#L24

Reply:
```suggestion
  skip: true  # [py<37 or s390x]
```

Reply:
Yes, I'll change it but first I want to rebuild `lazrs-python` https://github.com/AnacondaRecipes/lazrs-python-feedstock/pull/1 where this `py<39` was introduced by mistake, IMO

Reply:
done